### PR TITLE
Added option to interpret water meter in tenths

### PR DESCRIPTION
**Pull request by pasyn**
Added option to interpret water meter in tenths as well as a few other minor improvements listed below:
- Changed default unit for electricity meters to kWh as this is what most meters report.
- RTLTCP now has its own debug toggle.
- Made debug output easier to read.

#15

### CODE REVIEW

File: SDRMR/config.yaml [hunk 1]
```diff
@@ -22,24 +22,28 @@ arch:
...
+  rtltcpdebug: False
   msgType: scm
   ids: ""
   duration: 15
   pause_time: 30
   gas_unit_of_measurement: ft³
-  electric_unit_of_measurement: Wh
+  electric_unit_of_measurement: kWh
   water_unit_of_measurement: gal
+  water_use_tenths: True
```
Comment (cpyarger) [inline SDRMR/config.yaml:33]: Should default to false, or have a list or divisor,

File: SDRMR/config.yaml [hunk 2]
```diff
@@ -22,24 +22,28 @@ arch:
...
 options:
   debug: False
+  rtltcpdebug: False
   msgType: scm
   ids: ""
   duration: 15
   pause_time: 30
   gas_unit_of_measurement: ft³
-  electric_unit_of_measurement: Wh
+  electric_unit_of_measurement: kWh
```
Comment (cpyarger) [inline SDRMR/config.yaml:31]: By my understanding most meters report in kWh, so I am unsure why we are changing the default?

**Review by cpyarger — COMMENTED** [review]
Look over the comments, also, don't forget to update the version number so it pushes out to those using the addon.

File: SDRMR/config.yaml (hunk 2 above)
Comment (pasyn) [reply SDRMR/config.yaml:31]: The previous default was Wh not kWh. Like you said, most meters report in kWh

**Comment by cpyarger** [comment]
Fixes #15

**Pull request merged**